### ISSUE TITLE
Add statistical test of the coefficients with p-values from `lmac()`

### ISSUE THREAD

**Issue by isaactpetersen** [matloff/regtools#36]
It would be helpful to add a statistical test of the coefficients with p-values from `lmac()`.

**Comment by matloff** [comment]
Unfortunately--actually fortunately--I'm one of those people who strongly oppose the use of -values. See https://github.com/matloff/regtools/blob/master/inst/NoPVals.md

**Comment by isaactpetersen** [comment]
Thanks for the response, and I completely respect that. However, I'm not seeing any estimates of the uncertainty around the point estimates. For instance, would it be possible to get standard errors? Or perhaps confidence intervals from the bootstrapped estimates?

**Comment by matloff** [comment]
Yes, you can certainly get CIs and/or standard errors from the bootstrapped resamplings. Or, I have a jackknife function in my exNorm repo.
